EIPS/eip-2982.md

```diff
@@ -1,25 +1,21 @@
 ---
```

Comment:
A lot of the links have actually rotted. Would you mind going through the ones that haven't and shove them in the assets directory?

Reply:
/me feels vindicated